trio/_core/_run.py

```diff
@@ -1783,9 +1783,11 @@ def run(
         ):
             try:
                 with closing(runner):
-                    # The main reason this is split off into its own function
-                    # is just to get rid of this extra indentation.
-                    run_impl(runner, async_fn, args)
+                    with runner.entry_queue.wakeup.wakeup_on_signals():
+                        # The main reason this is split off into its own
+                        # function is just to get rid of this extra
+                        # indentation.
+                        run_impl(runner, async_fn, args)
```

Comment:
Wrapping a comment about indentation width to a third line because of the column limit? I wish the world already switched to 120 characters instead of the width of 1980's graphics adapters and terminals.